style.css

```diff
@@ -0,0 +1,21 @@
+body{
+    background-color: darkblue;
+
+}
```

Comment:
+Please use classes instead of IDs for repeated styles since it is easier and better practice. 
+Please use flexbox and css grid to layout the content  on your website. I